src/devices.cpp

```diff
@@ -0,0 +1,41 @@
+#include "devices.hpp"
+
+#include <Arduino.h>
+
+extern File file;
+extern MMA8452Q accelerometer;
+extern BMP280 bmp;
+extern Adafruit_GPS gps;
+
+namespace obc {
+
+void init()
+{
+    sd_init().expect("SD init failure");
```

Comment:
skrzywienie z google mock ale ten expect trochę sugeruje że fail jest czymś pożądanym :D

Reply:
tu jest background nazwy
https://stackoverflow.com/questions/66362625/why-is-rusts-expect-called-expect

tu jest dyskusja ludzi, dla których ona nie jest wystarczająco zrozumiala
https://github.com/rust-lang/rust/issues/35083

Ja sie do niej przyzwyczailem, wiec "pozyczyłem" ją z rusta bez zastanowienia, aczkolwiek nie jest to dla nas zaden problem zeby ją zmienić. Do przedyskutowania.